dataland-frontend/src/components/resources/frameworkDataSearch/ThreeLayerDataTable.vue

```diff
@@ -37,10 +37,12 @@
         </div>
         <div v-show="isExpanded(index)">
           <TwoLayerDataTable
+            data-test="TwoLayerTest"
             :arrayOfKpiDataObjects="arrayOfKpiDataObjectsMapItem[1]"
```

Comment:
Since i had to hover I couldn't take a proper screenshot. The tooltip formatting is not consistent with the "newer" fields seemingly not having the normal dark callout box formatting applied to them:

![image](https://github.com/d-fine/Dataland/assets/35459207/ab82fc36-dc02-4025-8363-4a2217f6548d)